core/src/main/scala/cats/data/IorT.scala

```diff
@@ -0,0 +1,509 @@
+   * scala> import cats.implicits._
+   * scala> IorT.right[String](Option(3))
+   * res0: cats.data.IorT[Option,String,Int] = IorT(Some(Right(3)))
+   * }}}
+   */
+  final def right[A]: RightPartiallyApplied[A] = new RightPartiallyApplied[A]
+
+  /**
+   * Alias for [[pure]]
+   * {{{
+   * scala> import cats.data.IorT
+   * scala> import cats.implicits._
+   * scala> IorT.rightT[Option, String](3)
+   * res0: cats.data.IorT[Option,String,Int] = IorT(Some(Right(3)))
+   * }}}
+   */
+  final def rightT[F[_], A]: PurePartiallyApplied[F, A] = pure
+
+  /**
+   * Creates a both version of `IorT[F, A, B]` from a `F[A]` and a `F[B]`
+   * {{{
+   * scala> import cats.data.IorT
+   * scala> import cats.implicits._
+   * scala> IorT.both(Option("err"), Option(3))
+   * res0: cats.data.IorT[Option,String,Int] = IorT(Some(Both(err,3)))
+   * }}}
+   */
+  final def both[F[_], A, B](fa: F[A], fb: F[B])(implicit F: Apply[F]): IorT[F, A, B] =
+    IorT(F.map2(fa, fb)((a, b) => Ior.Both(a, b)))
+
+  /**
+   * Uses the [[http://typelevel.org/cats/guidelines.html#partially-applied-type-params Partially Applied Type Params technique]] for ergonomics.
+   */
+  private[data] final class BothTPartiallyApplied[F[_]](val dummy: Boolean = true) extends AnyVal {
+    def apply[A, B](a: A, b: B)(implicit F: Applicative[F]): IorT[F, A, B] = IorT(F.pure(Ior.Both(a, b)))
+  }
+
+  /**
+   * Creates a both version of `IorT[F, A, B]` from a `A` and a `B`
+   * {{{
+   * scala> import cats.data.IorT
+   * scala> import cats.implicits._
+   * scala> IorT.bothT[Option]("err", 3)
+   * res0: cats.data.IorT[Option,String,Int] = IorT(Some(Both(err,3)))
+   * }}}
+   */
+  final def bothT[F[_]]: BothTPartiallyApplied[F] = new BothTPartiallyApplied[F]
+
+  /**
+   * Uses the [[http://typelevel.org/cats/guidelines.html#partially-applied-type-params Partially Applied Type Params technique]] for ergonomics.
+   */
+  private[data] final class PurePartiallyApplied[F[_], A](val dummy: Boolean = true) extends AnyVal {
+    def apply[B](b: B)(implicit F: Applicative[F]): IorT[F, A, B] = IorT(F.pure(Ior.right(b)))
+  }
+
+  /**
+   * Creates a right version of `IorT[F, A, B]` from a `B`
+   * {{{
+   * scala> import cats.data.IorT
+   * scala> import cats.implicits._
+   * scala> IorT.pure[Option, String](3)
+   * res0: cats.data.IorT[Option,String,Int] = IorT(Some(Right(3)))
+   * }}}
+   */
+  final def pure[F[_], A]: PurePartiallyApplied[F, A] = new PurePartiallyApplied[F, A]
+
+  /**
+   * Alias for [[right]]
+   * {{{
+   * scala> import cats.data.IorT
+   * scala> import cats.implicits._
+   * scala> val o: Option[Int] = Some(3)
+   * scala> val n: Option[Int] = None
+   * scala> IorT.liftF(o)
+   * res0: cats.data.IorT[Option,Nothing,Int] = IorT(Some(Right(3)))
+   * scala> IorT.liftF(n)
+   * res1: cats.data.IorT[Option,Nothing,Int] = IorT(None)
+   * }}}
+   */
+  final def liftF[F[_], A, B](fb: F[B])(implicit F: Applicative[F]): IorT[F, A, B] = right(fb)
```

Comment:
do you mind also add `liftK` like this one https://github.com/SystemFw/cats/blob/4830e326acab64223fa642a2b39111665e590763/core/src/main/scala/cats/data/EitherT.scala#L365 ? If you don't have the time it's fine too, I can add one in a follow up PR.

Reply:
sure, no problem